server/src/workflow-management/selector.ts

```
@@ -206,56 +276,65 @@ export const getElementInformation = async (
+                depth++;
               }
-
+          
               return deepest;
             };
-
-            // Handle iframe traversal
+          
+            const isInFrameset = () => {
+              let node = element;
+              while (node && node.parentElement) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)